images/spire/main.tf

```diff
@@ -26,7 +26,7 @@ module "latest-dev" {
   for_each = local.components
   source   = "../../tflib/publisher"
 
-  target_repository = var.target_repository
+  target_repository = "${var.target_repository}-${each.key}"
```

Comment:
Thanks for fixing this, it was wrong before, but I think it's still wrong for the new component?